index.rst

```diff
@@ -160,6 +160,13 @@ special characters for wildcard matching:
      - any integer numbers between ``num1`` and ``num2``, where ``num1`` and ``num2``
        can be either positive or negative
 
+If there is a path separator (``/``) at in the glob, then the glob is relative
+to the directory level of the particular `.editorconfig` file itself.
+Otherwise the pattern may also match at any level below the `.editorconfig`
+level. For example, ``*.c`` matches any file that ends with ``.c`` in the
+directory of ``.editorconfig``, but ``subdir/*.c`` only matches files that end
+with ``.c`` in the ``subdir`` directory in the directory of ``.editorconfig``.
+
```

Comment:
@xuhdev You mentioned

> we have been unclear with slashes at the end

That is mentioned in https://github.com/editorconfig/editorconfig/issues/493 .  Would you be willing to fix that as part of this PR as well?  https://github.com/editorconfig/editorconfig/issues/493#issue-1977582345 has some proposed text.  (If you would rather it be in a separate PR, that's fine.)

Reply:
(Let's do it in a separate PR)